ext/posix/posix.c

```diff
@@ -476,6 +476,9 @@ PHP_FUNCTION(posix_ttyname)
 	}
 #if defined(ZTS) && defined(HAVE_TTYNAME_R) && defined(_SC_TTY_NAME_MAX)
 	buflen = sysconf(_SC_TTY_NAME_MAX);
+#if defined(__FreeBSD__) && defined(_SC_PAGESIZE)
+        if (buflen < 1) buflen = sysconf(_SC_PAGESIZE);
```

Comment:
we tabs, not spaces...

Reply:
Checking why editorconfig didn't pick up properly.